### DISCUSSION TITLE
Change text for the prompt when giving the choice to teleport to a region or not.

### DISCUSSION THREAD

**Discussion by ichigoteguz** [foundryvtt/foundryvtt#11568] [Ideas]
Currently when using the teleport token behavior with regions, when you enable the "Allow Choice" option for the behavior, the text for the prompt is very meta, spelling out what region in what scene you are teleporting to, which the players might not even have that information at that point, it would be nice if we could customize this text.
For example instead of "Do you want to teleport PC to Waterfall Basin in BBG's Magic Forest?" it'd be nice if we could change it to "Attempt to jump down?".

**Comment by aaclayton** [comment]
Better as an issue.
